bazel/repository_locations.bzl

```diff
@@ -623,11 +623,11 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Go rules for Bazel",
         project_desc = "Bazel rules for the Go language",
         project_url = "https://github.com/bazelbuild/rules_go",
-        version = "0.27.0",
-        sha256 = "69de5c704a05ff37862f7e0f5534d4f479418afc21806c887db544a316f3cb6b",
-        urls = ["https://github.com/bazelbuild/rules_go/releases/download/v{version}/rules_go-v{version}.tar.gz"],
+        version = "0.28.0",
+        sha256 = "8e968b5fcea1d2d64071872b12737bbb5514524ee5f0a4f54f5920266c261acb",
+        urls = ["https://github.com/bazelbuild/rules_go/releases/download/v{version}/rules_go-v{version}.zip"],
         use_category = ["build", "api"],
-        release_date = "2021-03-17",
+        release_date = "2021-07-06",
```

Comment:
This release date might be `2021-07-07` but Github have changed the metadata in their HTML so if it is passing CI it should be OK

Reply:
seems like ci wants 06 https://dev.azure.com/cncf/envoy/_build/results?buildId=84482&view=logs&jobId=1c04237c-5f52-5959-5f83-5522b3556331&j=1c04237c-5f52-5959-5f83-5522b3556331&t=db3f3889-c922-566b-fa2e-4bc4948bc467